src/stats/show_stats.py

```diff
@@ -51,12 +55,14 @@ async def show_stats(ctx, stat_id):
             db.remove_single_show_stats_cache(stat_id)
 
     if stat_type == PERCENTAGE_BAR:
-        options, answers = get_options_and_their_counts(survey_question_id)
+        options, answers = get_options_and_their_counts(
+            survey_question_id, show_only_first_x_options_from_stat.get(stat_id)
+        )
 
         graph_size = (
             # these graphs have verbose answer_options therefore graphs need to be wider
             (25, 10)
-            if stat_id in ("7", "8", "9")
+            if stat_id in ("7", "8", "9", "10", "11")
```

Comment:
Let's move this to a constant (e.g. `WIDE_GRAPH_STAT_IDS`) in `available_stats.py`.

Reply:
Done